ROADMAP.md

```diff
@@ -8,124 +7,164 @@ Notes: This document is still in review will may be heavily modified based on st
-- [ ] Spec is merged and classified as "Candidate Recommendation"
-- [ ] (by virtue of the above) At least one major reference implementation exists
-- [ ] A well established testing criteria (defined at the outset of the project) is met (testing via Testground or otherwise)
-- [ ] Public documentation (on docs.libp2p) exists
+The "Definition of Done" for projects/sub-projects that involve writing new protocols/ modify existing ones usually consist of the following:
+- If a specification change is required:
+    - [ ] Spec is merged and classified as "Candidate Recommendation"
+    - [ ] (by virtue of the above) At least one major reference implementation exists
+- [ ] A well established testing criteria is met (defined at the outset of the project including but not limited to testing via Testground, compatibility tests with other implementations in the Release process, etc.)
+- [ ] Public documentation (on docs.libp2p.io) exists
+
+Supporting projects (such as testing or benchmarking) may have different criteria.
+
+## Benchmarking and Testing
+As mentioned in our [vision](#vision), performance and stability are core libp2p tenets. Rigorous benchmarking and testing help us uphold them. Related projects are listed in the [libp2p/test-plans roadmap](https://github.com/libp2p/test-plans/pull/56) and the [testground/testground roadmap](https://github.com/testground/testground/blob/master/ROADMAP.md). Our major priorities in Q4’22 and Q1’23 are:
+- [interoperability testing](https://github.com/libp2p/test-plans/issues/53) (across implementations & versions and between transports, muxers, & security protocols)
+- performance [benchmark go-libp2p using Testground](https://github.com/testground/testground/pull/1425) (create a benchmark suite to run in CI, create a public performance dashboard, [demonstrate libp2p is able to achieve performance on par with HTTP](https://github.com/libp2p/test-plans/issues/27))
 
-Others supporting projects (like testing or benchmarking) will have different criteria.
+These projects are parallel workstreams, weighed equally with roadmap items in this document. Some efforts like interoperability testing have a higher priority than implementation projects. The go-libp2p co-owns these efforts with the js-libp2p, rust-libp2p, and Testground/IPDX teams.
 
+[**Click here to see the shared Q4’22/Q1’23 testing and benchmarking priorities.**](https://github.com/libp2p/test-plans/pull/56)
 
 ## 🛣️ Milestones
 ### 2022
 
 #### Early Q4 (October)
--   A.1⚡Handshakes at the Speed of Light - **[Early Muxer Negotiation](https://github.com/libp2p/specs/issues/426)**
-- E.1 🧪 Future-proof testing - **RTT test for Early Muxer Negotiation**
-
-#### Mid Q4 (November)
--   F.1 ⏱ Future-proof Benchmarking - **[Using nix-builders](https://github.com/testground/testground/pull/1425)**
+- [B.1 ⚡ Early Muxer Negotiation](#1-early-muxer-negotiation)
```

Comment:
As mentioned in the "Benchmarking and Testing" section, the testing and benchmarking are in the test-plans roadmap, but if it is critical to add them here we can do this: https://github.com/libp2p/go-libp2p/blob/a89993a3ce1c6c44eb66d2cf05a36c6bfe323164/ROADMAP.md#mid-q4-november